ml/context/context.go

```diff
@@ -392,10 +394,17 @@ func GetParamOr[T any](ctx *Context, key string, defaultValue T) T {
 		return value
 	}
 
-	// Try converting, for instance, a float32 could be converted to float64.
+	// Try T.TextUnmarshaler(valueAny)
 	v := reflect.ValueOf(valueAny)
 	typeOfT := reflect.TypeOf(defaultValue)
-	if !v.CanConvert(typeOfT) {
+	valueT := reflect.New(typeOfT)
```

Comment:
Since you are here, would you mind adding one such conversion example in the `TestParams()` function, in `contest_test.go` file ?